pkg/controller/certificates/cleaner/cleaner.go

```diff
@@ -151,7 +151,7 @@ func isPendingPastDeadline(csr *capi.CertificateSigningRequest) bool {
 func isDeniedPastDeadline(csr *capi.CertificateSigningRequest) bool {
 	for _, c := range csr.Status.Conditions {
 		if c.Type == capi.CertificateDenied && isOlderThan(c.LastUpdateTime, deniedExpiration) {
-			klog.Infof("Cleaning CSR %q as it is more than %v old and denied.", csr.Name, deniedExpiration)
+			klog.InfoS("Cleaning CSR as it is past expiration and denied.", "csr", klog.KObj(csr), "deniedExpiration", deniedExpiration)
```

Comment:
```suggestion
		klog.InfoS("Cleaning CSR as it is past expiration and unhandled.", "csr", klog.KObj(csr), "age", time.Since(csr.CreationTime), "expiration": deniedExpiration, "condition", capi.CertificateDenied)
```

Reply:
I didn't use `"age", time.Since(csr.CreationTime)` as it throws this error: `csr.CreationTime undefined (type *"k8s.io/api/certificates/v1".CertificateSigningRequest has no field or method CreationTime)`.